frontend/model/contracts/group.js

```diff
@@ -1418,14 +1404,33 @@ sbp('chelonia/defineContract', {
           Vue.set(state, 'generalChatRoomId', data.chatRoomID)
         }
       },
-      sideEffect ({ contractID }, { state }) {
+      sideEffect ({ contractID, data }, { state }) {
         if (Object.keys(state.chatRooms).length === 1) {
           // NOTE: only general chatroom exists, meaning group has just been created
           sbp('state/vuex/commit', 'setCurrentChatRoomId', {
             groupID: contractID,
             chatRoomID: state.generalChatRoomId
           })
         }
+        // If it's the #General chatroom being added, add ourselves to it
+        if (data.chatRoomID === state.generalChatRoomId) {
+          sbp('chelonia/queueInvocation', contractID, () => {
+            const { identityContractID } = sbp('state/vuex/state').loggedIn
+            if (
+              state.profiles?.[identityContractID]?.status === PROFILE_STATUS.ACTIVE &&
+                      state.chatRooms?.[contractID]?.members[identityContractID]?.status !== PROFILE_STATUS.ACTIVE
```

Comment:
Ideally, there should be no extra indentation here on line 1421. If standardjs cmplains, try putting && on line 1421 at the start.